tests/e2e/esm/index.test.ts

```diff
@@ -382,7 +366,7 @@ describe('e2e esm tests', async () => {
 
     it('should be able to get balances of a specific to first page', async () => {
       const balances = await ario.getBalances({
-        cursor: 1000000,
+        cursor: '1000000',
```

Comment:
im a little confused by this test - this cursor seems configured to balances nots addresses?

Reply:
Balances are a valid cursor, but the type type expects a string.